security/botan/Portfile

```diff
@@ -23,20 +23,21 @@ long_description    Botan is a C++ library implementing a variety of \
 homepage            https://botan.randombit.net
 master_sites        ${homepage}/releases/
 distname            Botan-${version}
-extract.suffix      .tgz
+extract.suffix      .tar.xz
 
-checksums           rmd160  f3a4c1b963b47d543430f8705d7db87c64e013b9 \
-                    sha256  88481997578c27924724fea76610d43d9f59c99edfe561d41803bbc98871ad31 \
-                    size    7240021
+checksums           rmd160  d4c98a021d8519660538269c5713550bbabd2cd5 \
+                    sha256  541a3b13f1b9d30f977c6c1ae4c7bfdfda763cda6e44de807369dce79f42307e \
+                    size    5996928
 
-depends_build       port:python27
-depends_lib         path:lib/libssl.dylib:openssl port:zlib port:bzip2
+depends_build       port:python39
+depends_lib         port:bzip2 \
+                    port:lzma \
+                    path:lib/libssl.dylib:openssl \
```

Comment:
We should be using the `openssl` portgroup going forward instead of directly specifying OpenSSL as a dependency:

```
PortGroup openssl 1.0
```

Adding that portgroup means you can remove this dependency specification (`path:lib/libssl.dylib:openssl`).

The `openssl` portgroup builds against OpenSSL 3.0 by default, but if you need it to build against v1.1, you can do:

```
openssl.branch 1.1
```

Reply:
Thanks. Change done. There are deprecation warnings `OSSL_DEPRECATEDIN_3_0`but it does compile.